<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор власних чисел та власних векторів</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Введіть розмір квадратичної матриці</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>8</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
               <property name="displayIntegerBase">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>37</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>90</number>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>37</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Заповнити випадково</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Прийняти заповнення</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="method1RadioButton">
         <property name="text">
          <string>Метод Крилова</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="method2RadioButton">
         <property name="text">
          <string>Метод Леверрье-Федєєва</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Обрахувати</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="showPolynom">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вивести характеристичний графік полінома</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>881</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="polynom">
       <property name="text">
        <string>Тут буде записаний поліном</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="roots">
       <property name="text">
        <string>Тут будуть записані власні числа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vectors">
       <property name="text">
        <string>Тут будуть записані власні вектори</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="counter">
       <property name="text">
        <string>Тут буде записана складність</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат роботи программи</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
